Automattic/wp-calypso
# Site migration: show progress during a migration

Note: This change is behind a feature flag and is part of an in-progress larger project. For context, see pbkcP4-8-p2.

#### Changes proposed in this Pull Request

* Adds a progress view to the migrate section, with a simple item list and progress bar.

<img width="1166" alt="image" src="https://user-images.githubusercontent.com/1647564/69952448-204edd00-14ef-11ea-9939-a38799a3cee3.png">


#### Testing instructions



* Check out this pull request and run Calypso locally, or view it on calypso.live with the flag tools/migrate.
* This PR requires D36080-code and a sandboxed API. See the testing instructions for running D36080-code.
* Viewing calypso.localhost:3000/migrate/, you should see a SiteSelector component allowing you to choose a site to migrate a Jetpack site into.
* When you select a site, you should see another SiteSelector component, listing all the Jetpack sites your account has access to.
* Selecting one of those sites should take you to a confirmation screen.
* Accept the confirmation. You should see a progress screen detailing the steps of the migration.
* When the migration is done you should see a message telling you it is complete.
